### PR TITLE
insert and list persons

### PR DESCRIPTION
**Pull request by Pilar89**
Base de datos personas

### CODE REVIEW

**Comment by Pilar89** [comment]
Ultimo cambio

**Comment by bedorlan** [comment]
change the file extension

File: src/bases_de_datos.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    cur.execute("select * from personas") 
+    datos = cur.fetchall()
+     
+    
+    for fila in datos: 
+        print(fila)
+        
+        
+        
+
```
Comment (bedorlan) [inline src/bases_de_datos.py]: too many whitspaces

File: src/bases_de_datos.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    cur = cnn.cursor() 
+    sql = """insert into personas (nombre,cedula,telefono)
+        values (%s, %s, %s)"""
+    datos = (nombre, cedula, telefono)
+    cur.execute(sql, datos) 
+    cnn.commit()
+    
+      
+    
+
```
Comment (bedorlan) [inline src/bases_de_datos.py]: too many whitespaces

File: src/bases_de_datos.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    cur.execute(sql, datos) 
+    cnn.commit()
+    
+      
+    
+     
+def crear_menu():
+    while True:
+        
+        menu = int(input("Usted tiene el siguiente menu \n1 para agrepar persona \n2 para mostrar personas \n0 para salir \nEscriba una opcion: "))
```
Comment (bedorlan) [inline src/bases_de_datos.py]: this line is too long. try using """

File: src/bases_de_datos.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import mysql.connector
+cnn = mysql.connector.connect(host = "localhost", 
+                              user = "pilar", 
+                              passwd = "pilar", 
+                              database = "database1")
+def consultar_personas():
```
Comment (bedorlan) [inline src/bases_de_datos.py:7]: at least a new line

**Review by bedorlan — APPROVED** [review]

**Pull request merged**